### PR TITLE
Remove dev tests from the PR tests

### PR DESCRIPTION
**Pull request by h-mayorquin**
As in the title. From the discussion on  https://github.com/catalystneuro/neuroconv/pull/1082#discussion_r1759632013

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,8 @@
 * Using in-house `GenericDataChunkIterator` [PR #1068](https://github.com/catalystneuro/neuroconv/pull/1068)
 
 ## Improvements
+* Do not run dev tests on PRs [PR #1092](https://github.com/catalystneuro/neuroconv/pull/1082)
```
Comment (pauladkisson) [inline CHANGELOG.md:11]: 1092 is showing up twice, also the link is wrong

**Review by pauladkisson — CHANGES_REQUESTED** [review]
I'd like to see a dispatch of deploy-tests.yml to ensure it's working properly.

File: CHANGELOG.md (hunk 1 above)
Comment (h-mayorquin) [reply CHANGELOG.md]: Fixed

**Comment by h-mayorquin** [comment]
> I'd like to see a dispatch of deploy-tests.yml to ensure it's working properly.

What do you think could show up that is not showing up here in the PR? I just removed them from deploy.test.

**Comment by pauladkisson** [comment]
> > I'd like to see a dispatch of deploy-tests.yml to ensure it's working properly.
> 
> What do you think could show up that is not showing up here in the PR? I just removed them from deploy.test.

You know what, this is a dumb request -- I forgot that the PR triggers this workflow.

**Review by pauladkisson — APPROVED** [review]

**Pull request merged**